plugins/view-resources/src/actions.ts

```diff
@@ -68,9 +74,11 @@ export async function getActions (
   derived: Ref<Class<Doc>> = core.class.Doc,
   mode: ViewContextType = 'context'
 ): Promise<Action[]> {
-  const actions: Action[] = await client.findAll(view.class.Action, {
-    'context.mode': mode
-  })
+  let actions: Action[] | undefined = allActions.get(mode)
+  if (actions === undefined) {
+    actions = client.getModel().findAllSync(view.class.Action, { 'context.mode': mode })
+    allActions.set(mode, actions)
```

Comment:
For what? Get values from model should be instant, i don't think that it's really improve perfomance

Reply:
It clone 250 actions every time for 1500 messages :-) in our meeting minutes